src/cc2olx/models.py

```diff
@@ -272,6 +273,25 @@ def get_resource_content(self, identifier):
                     )
                     raise
                 return "html", {"html": html}
+            elif 'web_resources' in str(res_filename):
+                if imghdr.what(str(res_filename)):
```

Comment:
So, this will be true if res_filename is of a type inclued in the list at https://docs.w3cub.com/python~3.8/library/imghdr/

I don't think EdX supports all these formats but I don't think it's worth created a contant list of valid types to compare against. So, let's leave as is unless you've got a better idea.

Reply:
Yes, David even I am not sure how many of the image types with EdX support, but I have seen that developers at EdX prefer to use libraries instead of static code so I used imghdr library.